### ISSUE TITLE
Dynamic Component issue

### ISSUE THREAD

**Issue by ezra2310** [angular/angular#23115]
Hi Dears,
Im currently doing an dynamic website. so i created an one component it render in different different layout for eg single, double, triple, parallel,etc.. like that it will show grid/form.
My problem some time is used call the same component in UI and as well as in Model Popup also. In that time im getting "ExpressionChangedAfterItHasBeenCheckedError" issue. 
Take Eg: In brower i called the component and rendered in Single layout, after that by button click i need do call the same component should be open in Model popup for some reference. While open im getting the issue Expression-changed error. How to resolve this. Please help me !!!

Im added the component in entry component all those stuff did. 

 **private _ClearAndAddPageComponent() {
    this.addPageControlContainer.clear();
    this.addPageControlContainer.detach();
    var comp = this._cfr.resolveComponentFactory(RenderPageComponent);
    var expComponent = this.addPageControlContainer.createComponent(comp);
    expComponent.instance.pageLayoutData = this.pageLayoutData;
    expComponent.instance._ref = expComponent;
  }**

The above is code wrote add the component in on each click of menu to add in Brower UI
 Im using object like this to render the component,
Page-Section[0] -> For single layout
Page-Section[0] & Section[1] -> For double layout
Page-Section[0] & Section[1]& Section[2] -> For triple layout
Page-Section[0] & Section[1] -> For Parallel layout

In bower i used to use the Page-Section[0] -> For single layout and In model popup i have use the same component with Page-Section[0] & Section[1]& Section[2] -> For triple layout

**Comment by ericmartinezr** [comment]
Hello, we reviewed this issue and determined that it doesn't fall into the bug report or feature request category. This issue tracker is not suitable for support requests, please repost your issue on [StackOverflow](http://stackoverflow.com/) using tag `angular`.

If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).
